Assignments/Week1/FirstMissingInteger.py

```diff
@@ -0,0 +1,8 @@
+class Solution:
+    def firstMissingPositive(self, A):
+        hashmap=[0]*len(A)
```

Comment:
You don't need this hashmap for finding the most optimized solution. This is a good solution but try alternatives too. Think of the arithmetic sum and see how it can be used here.